packages/create-react-native-library/templates/common/CONTRIBUTING.md

```diff
@@ -94,6 +94,7 @@ To edit the Objective-C or Swift files, open `example/ios/<%- project.name -%>Ex
 To edit the Java or Kotlin files, open `example/android` in Android studio and find the source files at `<%- project.identifier -%>` under `Android`.
 
 <% } -%>
+
```

Comment:
need to remove this new line to make prettier happy for generated files. in vs code you can choose "Save without formatting" in command palette if you have auto format turned on.